CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+### Uncategorized
+- adapt to eip-1193 provider changes ([#170](https://github.com/MetaMask/eth-json-rpc-filters/pull/170))
```

Comment:
```suggestion
### Changed
- **BREAKING:** Adapt to EIP-1193 provider changes by replacing the deprecated `sendAsync` method with the `request` method ([#170](https://github.com/MetaMask/eth-json-rpc-filters/pull/170))

Reply:
@legobeat, could you update the CHANGELOG with this instead? We didn’t add request or remove sendAsync; we simply replaced instances where sendAsync was called from providers with request.​⬤

Reply:
Thank you for clarifying, will do!